_snippets/source-control-environments/configure-git-in-n8n.md

```diff
@@ -1,4 +1,9 @@
 1. Go to **Settings** > **Environments**.
-1. In **Git repository URL** enter the SSH URL for your repository.
-1. n8n supports ED25519 and RSA public key algorithms. ED25519 is the default. Select **RSA** under **SSH Key** if your git host requires RSA.
-1. Copy the SSH key.
+1. Choose your connection method:
+   - **SSH**: In **Git repository URL** enter the SSH URL for your repository (e.g., `git@github.com:username/repo.git`).
+   - **HTTPS**: In **Git repository URL** enter the HTTPS URL for your repository (e.g., `https://github.com/username/repo.git`).
+1. Configure authentication based on your connection method:
+   - **For SSH**: n8n supports ED25519 and RSA public key algorithms. ED25519 is the default. Select **RSA** under **SSH Key** if your git host requires RSA. Copy the SSH key.
+   - **For HTTPS**: Enter your credentials:
+     - **Username**: Your Git provider username
+     - **Password/Token**: Your Personal Access Token (PAT) from your Git provider
```

Comment:
Let's only refer to Password in the docs, since all major providers no longer support username/password authentication with https.
```suggestion
     - **Token**: Your Personal Access Token (PAT) from your Git provider
```

Reply:
@konstantintieber Just to check, did you mean "Let's only refer to Token in the docs..."? Your suggested change seems to imply that, but wanted to double check.

Reply:
Yep, I mistyped.